lib/new_relixir/transaction.ex

```diff
@@ -5,6 +5,25 @@ defmodule NewRelixir.Transaction do
 
   alias NewRelixir.Collector
 
+  @doc """
+  Send an error to New Relic
+  """
+  @spec record_error(transaction :: binary, error :: {binary, binary}) :: :ok
+  def record_error(transaction, {type, message})
```

Comment:
do you want to keep both of these functions since they resolve to the same thing? They have different typespecs though so maybe it's reasonable to have both?